src/app/components/input.error.component.ts

```diff
@@ -7,19 +7,17 @@ import { HlmInputErrorDirective } from './spartan/ui-input-helm/src';
 	standalone: true,
 	imports: [HlmInputErrorDirective],
 	host: {
-		class: 'tw-block tw-mt-0 md:tw-mt-1 tw-min-h-[20px] tw-mb-4',
+		class: 'tw-block tw-mt-0 md:tw-mt-1 tw-min-h-[20px] tw-mb-4 overflow-visible',
```

Comment:
```tw-``` prefix missing

Reply:
wasn't used/needed anyway - i removed it